cmd/scriggo/main.go

```diff
@@ -195,6 +198,34 @@ var commands = map[string]func(){
 		}
 		exit(0)
 	},
+	"run": func() {
```

Comment:
Invoking

```
$ scriggo run index.html index2.html index3.html
```

or 

```
$ scriggo run index.html --some-option
```

Should be invalid; instead, it just ignores all the parameters after the file name (`index.html` in this case).

Reply:
Fixed in commit "review: fix too many file names".